packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -315,7 +315,7 @@ module.exports = {
         try {
           // Send an email to the user.
           await strapi.plugins['email'].services.email.send({
-            to: user.email,
+            to: user.attributes.email,
```

Comment:
Can you replace by `to: (user.toJSON ? user.toJSON() : user).email` please

Reply:
Yes, thanks for example. I got it;)